exoctk/contam_visibility/visibilityPA.py

```diff
@@ -356,7 +363,52 @@ def using_gtvt(ra, dec, instrument, ephFileName=None, output='bokeh'):
 
     # Adding lists to a table object
     table = Table([v3minnan, v3maxnan, paMinnan, paMaxnan, paNomnan, gdnan, mjdnan],\
-                  names=('#min_V3_PA', 'max_V3_PA','min_Aperture_PA',\
+                  names=('min_V3_PA', 'max_V3_PA','min_Aperture_PA',\
                          'max_Aperture_PA', 'nom_Aperture_PA', 'Gregorian', 'MJD'))
 
-    return paMin, paMax, gd, fig, table
+    # Getting bad PAs

```

Comment:
Same here, are you just uncommenting from a previous commit or is this new functionality?

Reply:
this is new functionality. the `using_gtvt()` function is the new function that calculates the visibility using the JWST GTVT @mfixstsci